tests/web/utility/test_retrieve_stats_from_import_results.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+
+from func_sig_registry.utils.abi import (
+    retrieve_stats_from_import_results,
+)
+
+@pytest.mark.parametrize(
+    'array_of_tuples,expected',
+    (
+        (
+            [(None, False),(None, True),None,(None, True),(None, False)], 
+            (5, 2, 2, 1)
+        ),
+        (#Empty Array
```

Comment:
:100: for labeling these test cases.  I like to think of this as "being kind to future me" since at some point you/me will be back here and will be happy to have this extra contextual information.